### DISCUSSION TITLE
Displaying the actual error in the client

### DISCUSSION THREAD

**Discussion by ccfiel** [hayes/pothos#1021] [Q&A] [answered]
When there is an error in executing a Graphql query or mutation on the client side, the error is always ```Unexpected error.```
You can only see the exact error on the server side. Is there a way to display the actual error on the client side? Please take a look at the image below.

![SCR-20230908-mprf](https://github.com/hayes/pothos/assets/1788360/8881861c-7b3d-47e4-8792-752008605321)

**Comment by hayes** [comment, marked as answer]
This is a feature of graphql yoga, you can read more about customizing this behavior here: https://the-guild.dev/graphql/yoga-server/docs/features/error-masking

**Comment by ccfiel** [comment]
@hayes as I read the link its a feature ```error-masking``` :)
